apps/omg_watcher/lib/block_getter/core.ex

```diff
@@ -250,7 +237,7 @@ defmodule OMG.Watcher.BlockGetter.Core do
       )
 
     blocks_numbers =
-      (potential_block_withholding_numbers ++ potential_next_block_numbers)
+      potential_next_block_numbers
```

Comment:
this is unexpected - we want the block numbers marked as potential withholding to be re-downloaded